vulcanai2/models/dnn.py

```diff
@@ -45,7 +52,7 @@ def _create_network(self):
                 pass
```

Comment:
@priyatharsan For the if statement above shouldn't it be `self.in_dim = self._input_network.dims[-1]` instead of `self.in_dim = self.dims[-1]` ?

Reply:
you are right. it should be `self._input_network.dims[-1]`

Reply:
ok push that commit/change to this PR once you change it